### ISSUE TITLE
Tuner doesn't seem to calculate note properly

### ISSUE THREAD

**Issue by kusog** [phenomnomnominal/tuner.coffee#2]
In my initial test, it seems to show the wrong note.  I am running a trusted tuner and using a simple recorder and see that this tuner is not showing the proper note.  Have you tested this actually works properly, not just runs and appears to work?

**Comment by phenomnomnominal** [comment]
Yes, I've used it for almost 2 years now. It's not perfect, and there's definitely room for improvement, particularly in some of the heuristics it uses for removing potential harmonics. If you could give some more details, I could have a look.

**Comment by kusog** [comment]
It appears to work if I put the recorder right near mic.  Is there a sensitivity option?

**Comment by kusog** [comment]
I want to integrate it into my code http://kusogmusic.com/compact

**Comment by phenomnomnominal** [comment]
There's possibly something that can be done with the noise removal stuff. Have a look at [the docs](http://phenomnomnominal.github.io/projects/tuner/docs/noiseRemoval.html). I'll have a fiddle later on, but it's a pretty hard one to get working, since stock computer microphones are mostly shit.
